frontend/modelarchive/src/main/java/com/amazonaws/ml/mms/archive/Manifest.java

```diff
@@ -190,20 +190,19 @@ public void setModelVersion(String modelVersion) {
             this.modelVersion = modelVersion;
         }
 
-        public String getParametersFile() {
-            return parametersFile;
+        public Map<String, Object> getExtensions() {
+            return extensions;
         }
 
-        public void setParametersFile(String parametersFile) {
-            this.parametersFile = parametersFile;
+        public void setExtensions(Map<String, Object> extensions) {
```

Comment:
Should we have a null check here as well ? and if a null extensions is passed, we set extensions to a new instance of LinkedHashMap ?

Reply:
The extensions is nullable. If extensions is null, we don't serialize this field into json. extensions will be created if addExtensions() is called at later time.